### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by pmoranga**
Fixed:
- lint tests
- travis.yml build
- Amazon installation

### CODE REVIEW

**Comment by vigneshsarma** [comment]
Since this one has the other pulls merged, closing them.

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ rvm:
   - ruby-head
 script:
   - "rake lint"
-  - "rake spec SPEC_OPTS='--format documentation'"
+#  - "rake spec SPEC_OPTS='--format documentation'"
```
Comment (vigneshsarma) [inline .travis.yml]: I just realized, this prevents the tests from running.

Comment (pmoranga) [inline .travis.yml]: in my travis it run all the tests.
https://travis-ci.org/pmoranga/puppetlabs-logentries

Comment (vigneshsarma) [inline .travis.yml]: It is only running `lint` in this https://travis-ci.org/pmoranga/puppetlabs-logentries/jobs/56733135 build for example. 
I have put the tests back in #5. 
https://travis-ci.org/Aplopio/puppetlabs-logentries/builds/56740220

**Pull request merged**